pkg/sidecar_mounter/sidecar_mounter_config_test.go

```diff
@@ -144,6 +144,18 @@ func TestPrepareMountArgs(t *testing.T) {
 			expectedArgs:          defaultFlagMap,
 			expectedConfigMapArgs: defaultConfigFileFlagMap,
 		},
+		{
+			name: "temporary until we support profile flag - should return valid args with error correctly",
+			mc: &MountConfig{
+				BucketName: "test-bucket",
+				BufferDir:  "test-buffer-dir",
+				CacheDir:   "test-cache-dir",
+				ConfigFile: "test-config-file",
+				Options:    []string{"profile=true", "profile:true"},
```

Comment:
nit: Profile is a profile (e.g. aiml-training, not a boolean)

Also I don't understand : "should return valid args with error correctly" what does error correctly mean? I think we can saw should return valid args with profile removed when specified, or something